vispy/gloo/glir.py

```diff
@@ -876,18 +905,22 @@ def set_interpolation(self, min, mag):
 class GlirTexture2D(GlirTexture):
     _target = gl.GL_TEXTURE_2D
     
-    def set_size(self, shape, format):
+    def set_size(self, shape, format, internalformat):
```

Comment:
It depends on how we define the GLIR command. If we define it as allowing internalformat to be omitted, we need a default value. I think I prefer that we define commands as fully specified, with optional arguments set to None though. @rossant WDYT?